receiver/memcachedreceiver/testdata/expected_metrics/test_scraper/exepected.json

```diff
@@ -0,0 +1,389 @@
+{
+   "resourceMetrics": [
+      {
+         "instrumentationLibraryMetrics": [
+            {
+               "instrumentationLibrary": {
+                  "name": "otelcol/memcached"
+               },
+               "metrics": [
+                  {
+                     "description": "Commands executed.",
+                     "name": "memcached.commands",
+                     "sum": {
+                        "aggregationTemporality": "AGGREGATION_TEMPORALITY_CUMULATIVE",
+                        "dataPoints": [
+                           {
+                              "asInt": "0",
```

Comment:
Since we have full control of the fake client, let's pass in non-zero values, to ensure that they are parsed and set appropriately.

Reply:
Updated